src/watchmaker/__init__.py

```diff
@@ -286,11 +284,11 @@ def _get_config(self):
                     config[worker_name]['__merged'] = False
                 if not config[worker_name].get('__merged'):
                     # Merge worker_args into config params
-                    config[worker_name]['Parameters'].update(self.worker_args)
+                    config[worker_name]['config'].update(self.worker_args)
                     config[worker_name]['__merged'] = True
-            except KeyError:
+            except Exception:
                 msg = (
-                    'Malformed worker, missing "Parameters" key; worker = {0}'
+                    'Failed to merge worker config; worker={0}'
```

Comment:
From last PR, I noticed that you changed any {}.format to using %s - so I assume you want consistency and just missed this one.